src/components/Header.jsx

```diff
@@ -0,0 +1,20 @@
+// Imports
+
+
+// Will return a "container" for the header of the website
+function Header() {
+    return (
+
+        <>
+            <div id="Header">
+              <img id="Logo" src={ require("../pictures/logo.png") } alt="Balanced"/>
+            </div>
+        </>
```

Comment:
Remove "<>" and "</>". 

This is being replaced with the "<div id="Header"...>" and "</div>".

In addition to the div header, replace it with the tags "<header>" and "</header>" just for for semantics